repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/cihub.ftl

```diff
@@ -0,0 +1,26 @@
+<#ftl output_format="HTML">
+<#include "../include/imports.ftl">
+<section class="document-content">
+    <h1 data-uipath="ps.document.title">Clinical Indicators</h1>
+    <p data-uipath="ps.document.content">
+        This is a collection of over a thousand datasets that we publish. The information
+        is for clinical staff, commissioners, researchers, and others needing data and
+        evidence to help with decision-making in health and care.
```

Comment:
Does this text want to be configurable throgh a field in the CMS instead of hardcoded in the template? I can imagine that someone will want the text tweaking at some point.